panic: runtime error: slice bounds out of range [:3] with capacity 2

Have error with usage multiple path in to --config.paths

I using last version binary package

`# ./cgroup_exporter --version
cgroup_exporter, version 1.1.0 (branch: HEAD, revision: 78f8b7117673c85644d58b302b520c98b3011c16)
  build user:       goreleaser
  build date:       2026-02-11T17:37:36Z
  go version:       go1.25.7
  platform:         linux/amd64
  tags:             unknown`

I'm trying to monitor cgroups for lxc containers 

`# ./cgroup_exporter --path.cgroup.root="/sys/fs/cgroup" --config.paths=/lxc.payload.host1,/lxc.payload.host2,/lxc.monitor.host1,/lxc.monitor.host2 --log.level=info
time=2026-02-24T09:08:04.355Z level=INFO source=cgroup_exporter.go:74 msg="Starting cgroup_exporter" version="(version=1.1.0, branch=HEAD, revision=78f8b7117673c85644d58b302b520c98b3011c16)"
time=2026-02-24T09:08:04.355Z level=INFO source=cgroup_exporter.go:75 msg="Build context" build_context="(go=go1.25.7, platform=linux/amd64, user=goreleaser, date=2026-02-11T17:37:36Z, tags=unknown)"
time=2026-02-24T09:08:04.355Z level=INFO source=cgroup_exporter.go:76 msg="Starting Server" address=:9306



time=2026-02-24T09:08:12.021Z level=ERROR source=cgroupv2.go:262 msg="Unable to get PIDs for name" name=/lxc.payload.host1/init.scope
time=2026-02-24T09:08:12.021Z level=ERROR source=cgroupv2.go:262 msg="Unable to get PIDs for name" name=/lxc.payload.host1/system.slice
panic: runtime error: slice bounds out of range [:3] with capacity 2

goroutine 40 [running]:
github.com/treydock/cgroup_exporter/collector.getNamev2({0xc00031a063, 0x29}, {0x7ffd6c5cc655, 0x29}, 0xc00011e3a0)
        /work/collector/cgroupv2.go:107 +0x2e5
github.com/treydock/cgroup_exporter/collector.(*Exporter).collectv2(0xc000176000)
        /work/collector/cgroupv2.go:236 +0xe07
github.com/treydock/cgroup_exporter/collector.(*Exporter).Collect(0xc000176000, 0xc0001562a0)
        /work/collector/collector.go:169 +0x3b
github.com/prometheus/client_golang/prometheus.(*Registry).Gather.func1()
        /root/go/pkg/mod/github.com/prometheus/client_golang@v1.23.2/prometheus/registry.go:456 +0x105
created by github.com/prometheus/client_golang/prometheus.(*Registry).Gather in goroutine 34
        /root/go/pkg/mod/github.com/prometheus/client_golang@v1.23.2/prometheus/registry.go:548 +0xbef`

help pls fix error 
`panic: runtime error: slice bounds out of range [:3] with capacity 2
`

--- a/.github/workflows/test.yaml
+++ b/.github/workflows/test.yaml
@@ -26,10 +26,10 @@ jobs:
       - name: Run coverage
         run: make coverage
       - name: Upload coverage to Codecov
-        uses: codecov/codecov-action@v4
+        uses: codecov/codecov-action@v5
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
-          file: ./coverage.txt
+          files: ./coverage.txt
   build:
     runs-on: ubuntu-latest
     name: Build

--- a/collector/cgroupv2.go
+++ b/collector/cgroupv2.go
@@ -100,13 +100,15 @@ func getInfov2(name string, pids []int, metric *CgroupMetric, logger *slog.Logge
 func getNamev2(pidPath string, path string, logger *slog.Logger) string {
 	dirs := strings.Split(pidPath, "/")
 	var name string
+	endIndex := 3
 	if strings.Contains(path, "slurm") {
-		keepDirs := dirs[0:4]
-		name = strings.Join(keepDirs, "/")
-	} else {
-		keepDirs := dirs[0:3]
-		name = strings.Join(keepDirs, "/")
+		endIndex = 4
 	}
+	if len(dirs) < endIndex {
+		endIndex = len(dirs)
+	}
+	keepDirs := dirs[0:endIndex]
+	name = strings.Join(keepDirs, "/")
 	logger.Debug("Get name from path", "name", name, "pidPath", pidPath, "path", path, "dirs", fmt.Sprintf("+%v", dirs))
 	return name
 }

